great_expectations/render/renderer/v3/suite_profile_notebook_renderer.py

```diff
@@ -192,7 +192,11 @@ def _add_user_configurable_profiler_cells(self) -> None:
     # },
     # column_value_nonnullity_rule={
     # },
-    # numeric_columns_rule={
+    # numeric_columns_low_precision_rule={
+    #     "false_positive_rate": 0.1,
+    #     "random_seed": 43792,
+    # },
+    # numeric_columns_high_precision_rule={
```

Comment:
@NathanFarmer I was hoping that you put in something about rounding (like your defaults -- so that users would see the difference between low and high precision and modify as they deem appropriate).  Thanks!